test/release/examples/benchmarks/shootout/knucleotide.chpl

```diff
@@ -15,7 +15,7 @@ proc main(args: [] string) {
   // Open stdin and a binary reader channel
   const consoleIn = new file(0),
         fileLen = consoleIn.size,
-        stdinNoLock = consoleIn.reader(kind=ionative, locking=false);
+        stdinNoLock = consoleIn.reader(locking=false);
```

Comment:
No `binaryDeserializer` required here?

Reply:
This test mainly uses ``readLine``, which reads byte-by-byte directly and isn't sensitive to ``iokind``